controller/templates/controller/partials/state_machine.html

```diff
@@ -0,0 +1,10 @@
+{% load render_table from django_tables2 %}
+{% for event in events %}<input type="hidden" name="event" id="event_{{ event }}" value="{{ event }}">{% endfor %}
```

Comment:
I'm found it a little difficult to puzzle out how this worked. Could we not use the same approach as the action buttons in the process manager. Keeps the behaviour attached to the buttons that are clicked.

Reply:
Simplified